nixos/modules/installer/cd-dvd/installation-cd-graphical-plasma5-new-kernel.nix

```diff
@@ -3,5 +3,7 @@
 {
   imports = [ ./installation-cd-graphical-plasma5.nix ];
 
+  isoImage.edition = "plasma5-new-kernel";
```

Comment:
I think the whole volumeID is still limited to 32 characters, not sure what happens if we exceed this, either the ISO won't build or we lose (part of) the architecture.

Reply:
Yeah, I don't think there's anything to get around it being limited to exactly 32 bytes https://wiki.osdev.org/ISO_9660#The_Primary_Volume_Descriptor. I wonder what exactly I can do.

Reply:
I did https://github.com/NixOS/nixpkgs/pull/83551/commits/69081e3dc742d2f7a617e216226c4c5eb36c51c0